### PR TITLE
🌐 Add Chinese translation for Tutorial - Response Status Code

### PR DESCRIPTION
**Pull request by waynerv**
@RunningIkkyu @maoyibo @yanever Please take a look.

### CODE REVIEW

**Review by yanever — APPROVED** [review]

**Review by maoyibo — COMMENTED** [review]
Looks good to me

File: docs/zh/docs/tutorial/response-status-code.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+* **`300`** 及以上状态码用于「重定向」。具有这些状态码的响应可能有或者可能没有响应体，但 `304`「未修改」是个例外，该响应不得含有响应体。
+* **`400`** 及以上状态码用于「客户端错误」响应。这些可能是你第二常使用的类型。
+    * 一个例子是 `404`，用于「未找到」响应。
+    * 对于来自客户端的一般错误，你可以只使用 `400`。
+* `500` 及以上状态码用于服务器端错误。你几乎永远不会直接使用它们。当你的应用程序代码或服务器中的某些部分出现问题时，它将自动返回这些状态代码之一。
+
+!!! tip
+    要了解有关每个状态代码以及适用场景的更多信息，请查看 <a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Status" class="external-link" target="_blank"><abbr title="Mozilla Developer Network">MDN</abbr> 关于 HTTP 状态码的文档</a>。
+
+## 记住名称的捷径
```
[the remark this answers is not in the document]
Comment (waynerv) [reply docs/zh/docs/tutorial/response-status-code.md:60]: I think “捷径” is more concisely here in a title, and their meaning is almost the same.

**Review by Laineyzhang55 — APPROVED** [review]
Good translation.

**Comment by tiangolo** [comment]
Awesome! Thank you @waynerv ! :rocket: :bow: 

And thanks @maoyibo , @yanever , and @Laineyzhang55 for the reviews! :coffee: :cake:

**Pull request merged**